src/compile/header/index.ts

```diff
@@ -145,27 +135,15 @@ function getSort(facetFieldDef: FacetFieldDef<string>): VgComparator {
   }
 }
 
-function getHeaderTitleProperties(config: Config) {
-  const props = {};
-  if (config.header) {
-    for (const prop of HEADER_TITLE_PROPERTIES) {
-      if (config.header[prop]) {
-        props[HEADER_TITLE_PROPERTIES_MAP[prop]] = {value: config.header[prop]};
-      }
-    }
-  }
-  return props;
-}
-
-function getHeaderGroup(model: Model, channel: HeaderChannel, headerType: HeaderType, layoutHeader: LayoutHeaderComponent, headerCmpt: HeaderComponent) {
+export function getHeaderGroup(model: Model, channel: HeaderChannel, headerType: HeaderType, layoutHeader: LayoutHeaderComponent, headerCmpt: HeaderComponent) {
   if (headerCmpt) {
     let title = null;
     const {facetFieldDef} = layoutHeader;
     if (facetFieldDef && headerCmpt.labels) {
       const {header = {}} = facetFieldDef;
       const {format, labelAngle} = header;
       const config = model.config? model.config : undefined;
-      const defaults = {
+      const update = {
```

Comment:
Do we really need to keep these `update` block?  Can't we use the angle, anchor, and baseline of the `title` block?